### ISSUE TITLE
BuyStop

### ISSUE THREAD

**Issue by Queracus** [EarnForex/PositionSizeCalculator#9]
2.33 version on mt4 allways opens Buy stop or Buy limit orders with totaly wrong numbers [https://prnt.sc/xyxplt](url) [https://prnt.sc/xyxrxb](url) executed by script

**Comment by EarnForex** [comment]
If you updated from a previous version of PSC, please remove old chart objects from the chart. Ctrl+B, delete the unprefixed StopLossLine, EntryLine, TakeProfitLine.

**Comment by Queracus** [comment]
it was a completely fresh instalation on fresh windows instalation. Reverted back to 2.31 since 2.32 also had problems

**Comment by EarnForex** [comment]
Unfortunately, I cannot reproduce this issue.
Perhaps, you did use a chart template from the previous version of PSC?
Also, which version of PSC-Trader script did that happen with? 1.10?

**Comment by Queracus** [comment]
script was taken from the EarnForex. says version 1.1

**Comment by frenchdna** [comment]
I had a smilar issue on MT5 with v2.33 and script 1.10. I came here, saw the comment about chart objects. I unloaded the indicator, then via the object list I removed the leftover lines, those starting with PSC_ were gone but I had some leftover (EntryLine, StopLine and TakeProfitLine) from a previous version probably so I removed those too. Then I reloaded the indicator onto the chart and now it works fine again.
